extensions/blockchains/ethereum/assets/0xA9299C296d7830A99414d1E5546F5171fA01E9c8/info.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "DGLD",
```

```suggestion
  "name": "Digital Gold",
```

or is DGLD preferable?

---

Good catch, updated